### PR TITLE
DAOS-11192 test: increase subprocess pattern_timeout

### PR DESCRIPTION
**Pull request by daltonbohning**
Test-tag: test_ior_crash
Test-repeat: 10
Skip-unit-tests: true
Skip-fault-injection-test: true

Increase IOR subprocess pattern_timeout to 60s.
Misc cleanup

Signed-off-by: Dalton Bohning <dalton.bohning@intel.com>

### CODE REVIEW

**Review by shimizukko — DISMISSED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Comment by daltonbohning** [comment]
@phender Suggesting using `SubProcessCommand` instead of `ExecutableCommand` - changing to draft

File: src/tests/ftest/util/ior_utils.py [hunk 1]
```diff
@@ -89,7 +86,7 @@ def __init__(self, namespace="/run/ior/*"):
         Args:
             namespace (str, optional): path to yaml parameters. Defaults to "/run/ior/*".
         """
-        super().__init__(namespace, "ior")
+        super().__init__(namespace, "ior", timeout=60)
```
Comment (phender) [inline src/tests/ftest/util/ior_utils.py:89]: Should this be `timeout=30` and then we add the following to ior/crash.yaml:
```
ior:
  pattern_timeout: 60
```

Comment (daltonbohning) [reply src/tests/ftest/util/ior_utils.py:89]: I was thinking that if ior/crash could take longer than 30 seconds, then other tests could take longer than 30 seconds. There isn't anything special about the way ior/crash uses IOR

Comment (phender) [reply src/tests/ftest/util/ior_utils.py:89]: Reasonable logic :)

Comment (daltonbohning) [reply src/tests/ftest/util/ior_utils.py:89]: FWIW, I think ior/crash is the only test using this function for IOR. Which is why we don't see the issue elsewhere

**Comment by daltonbohning** [comment]
Even a 60s timeout is not enough, so we probably need to address the core problem

**Review by shimizukko — APPROVED** [review]

**Review by phender — APPROVED** [review]

**Pull request merged**